templates/definition/charger/viessmann.yaml

```diff
@@ -60,6 +60,13 @@ params:
     required: true
     default: 45
     type: int
+  - name: watchdog
```

Comment:
This isn't particularly clear. Trigger what? Why don't we just hard-code a 30m interval below?

Reply:
I just though if it's under advanced it might be useful to some users. 
But I don't mind, just changed this to hard coded - does it look right?
https://github.com/evcc-io/evcc/pull/20111/commits/1ec099e3475b821cf7e87c6221857224e874f91d